### PR TITLE
Notepad example

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: examples/notepad/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+dependencies {
+    implementation(compose.desktop.currentOs)
+    implementation(compose.materialIconsExtended)
+}
+
+compose.desktop {
+    application {
+        mainClass = "MainKt"
+        jvmArgs("-Dskiko.rendering.laf.global=true")
```
Comment (olonho) [inline examples/notepad/build.gradle.kts:24]: So we no longer set it by default?

File: examples/notepad/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            packageVersion = "1.0.0"
+
+            windows {
+                menu = true
+                // see https://wixtoolset.org/documentation/manual/v3/howtos/general/generate_guids.html
+                upgradeUuid = "61DAB35E-17CB-43B0-81D5-B30E1C0830FA"
+            }
+
+            macOS {
+                // Use -Pcompose.desktop.mac.sign=true to sign and notarize.
```
Comment (olonho) [inline examples/notepad/build.gradle.kts:38]: Guess smth else shall be here.

File: examples/notepad/src/main/kotlin/util/AwtIcons.kt [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+import java.awt.Point
+import java.awt.image.BufferedImage
+import java.awt.image.ColorModel
+import java.awt.image.DataBuffer
+import java.awt.image.DataBufferInt
+import java.awt.image.Raster
+import java.awt.image.SinglePixelPackedSampleModel
+import kotlin.math.roundToInt
+
+suspend fun ImageVector.toAwtImage(tintColor: Color): BufferedImage {
```
Comment (olonho) [inline examples/notepad/src/main/kotlin/util/AwtIcons.kt:28]: Maybe add those(non-suspend versions) to https://github.com/JetBrains/skiko/blob/master/skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Convertors.kt

File: examples/notepad/src/main/kotlin/util/Compose.kt [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                ) {
+                    result = content()
+                }
+            }
+        }
+    }
+    return result as T
+}
+
+private object YieldFrameClock : MonotonicFrameClock {
```
Comment (olonho) [inline examples/notepad/src/main/kotlin/util/Compose.kt:40]: Do we really need those?

File: examples/notepad/build.gradle.kts (hunk 1 above)
Comment (igordmn) [reply examples/notepad/build.gradle.kts:24]: We don't set in by default in Compose, but can set it by default in our plugin (we never did it before).

@AlexeyTsvetkov, what you think?

File: examples/notepad/src/main/kotlin/util/AwtIcons.kt (hunk 3 above)
Comment (igordmn) [reply examples/notepad/src/main/kotlin/util/AwtIcons.kt:28]: ImageVector is a Compose class, so we can add it to Compose in the future

File: examples/notepad/src/main/kotlin/util/Compose.kt (hunk 4 above)
Comment (igordmn) [reply examples/notepad/src/main/kotlin/util/Compose.kt:40]: Compose.kt is needed for AwtIcons.kt, that allow us to convert built-in material icons to BufferedImage

File: examples/notepad/build.gradle.kts (hunk 2 above)
Comment (igordmn) [reply examples/notepad/build.gradle.kts]: Removed. we don't need it

File: examples/notepad/src/main/kotlin/util/AwtIcons.kt (hunk 3 above)
Comment (igordmn) [reply examples/notepad/src/main/kotlin/util/AwtIcons.kt:28]: We can replace part of this code by this:
```
    CanvasDrawScope().draw(
        density, layoutDirection, canvas, intrinsicSize
    ) {
        draw(intrinsicSize, colorFilter = colorFilter)
    }

    return bitmap.asDesktopBitmap().toBufferedImage()
```

Icon in Tray is ok, but icon in titlebar is not ok. Something is wrong in `toBufferedImage` (will figure out this later).

**Review by olonho — APPROVED** [review]

**Pull request merged**